### DISCUSSION TITLE
Flux variability analysis

### DISCUSSION THREAD

**Discussion by sophiez8** [micom-dev/micom#94] [Q&A] [answered]
Dear MICOM team,

is there a micom workflow for carrying out a flux variability analysis?

**Comment by cdiener** [comment, marked as answer]
Not yet, but it would make sense to add it. One limitation would be that it is probably unfeasible to run it for all fluxes (would take months for a single model), but it would be feasible selecting a small set of reactions (few hundreds or less).

**Reply by sophiez8** [reply]
thank you!
